scripts/tests/test.mk

```diff
@@ -14,6 +14,9 @@ test-unit: # Run your unit tests from scripts/test/unit @Testing
 test-lint: # Lint your code from scripts/test/lint @Testing
 	make _test name="lint"
 
+test-style: # Run your markdown files through a prose style checker @Testing
```

Comment:
I'm not certain that running the `vale` linter as part of `make test` is a good decision at this stage. There's a risk that this process will become very monolithic. Can we postpone adding it there, please? Instead, can we add it to the `pre-commit` hooks?

The intention is that we'll review the CI/CD pipeline structure in PR #123 and in any subsequent ones based on feedback. This will include all the `test-*` targets.

Reply:
Removed as discussed